ivy/functional/frontends/jax/numpy/manipulations.py

```diff
@@ -36,6 +36,11 @@ def concatenate(arrays, axis=0, dtype=None):
     return ret
 
 
+@to_ivy_arrays_and_back
+def repeat(a, repeats, axis=None, *, total_repeat_length=None):
+    return ivy.repeat(a, repeats, axis=axis)
```

Comment:
Handling of `total_repeat_length` is also needed, both here and in the test.

Reply:
@AnnaTz Thanks for the comment. I tried to add this before, but the test for it was failing, and when I removed it, the test passed.

`pytest ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_manipulation.py::test_jax_numpy_repeat`

**Error:**
`repeat() got an unexpected keyword argument 'total_repeat_length'`

Reply:
Did you try to pass it directly to `ivy.repeat`? I don't think we support this in our api currently. You would have to write a little extra code in you `repeat` function to handle it.

Reply:
please review the changes